lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -139,6 +147,44 @@ WebProcessingServiceCatalogFunction.parameterConverters = [
         }
     },
     {
+        id: 'RegionType',
+        inputToFunctionParameter: function(catalogFunction, input) {
+            if (!defined(input.LiteralData)) {
+                return undefined;
+            }
+
+            if (input.Identifier !== "regionType") {
```

Comment:
`Identifier` is basically the name of the parameter, right?  What if we want to select two region types?  That's not purely theoretical, the Spatial Detailing service does exactly that.